web/modules/custom/portland/portland.module

```diff
@@ -198,11 +213,8 @@ function portland_group_content_type_delete(GroupContentTypeInterface $groupCont
 
 /**
  * Implements hook_ENTITY_TYPE_presave().
+ * POWR-475 Changing short name of a group can cause the group URL to be different from the URLs in the sidebar menu. Root cause is that the group URL is set by PathAuto to be clean and unique. But the URL in sidebar menu is formed by Twig code using the short name as is. The solution is to update short name to ensure it's clean and unique when the group is updated.
```

Comment:
"sidebar menu" should be updated to "group menu". We moved that menu inline under group header information.

Reply:
Updated